tests/SimpleStringTest.cpp

```diff
@@ -257,6 +257,12 @@ TEST(SimpleString, countTogether)
     LONGS_EQUAL(4, str.count("ha"));
 }
 
+TEST(SimpleString, countEmptyString)
+{
+    SimpleString str("hahahaha");
+    LONGS_EQUAL(1, str.count(""));
```

Comment:
Returning zero would be fine by me and work okay. But I could't give you any better justification for returning zero than for returning 1...

After all, how many times is the empty string contained in a string ;-).